src/views/Document.vue

```diff
@@ -41,13 +38,19 @@
 import { mapGetters } from 'vuex';
 import ResultHeader from '../components/Result/ResultHeader';
 import Toggle from '@/components/Global/Toggle/Toggle';
+import { prevRoute } from '../router/index.js';
 
 export default {
 	components: {
 		ResultHeader,
 		Toggle
 	},
 	computed: {
+		backToPrev: function () {
+			if (prevRoute.name === null)
```

Comment:
Please, always use parenthesis when in condition, e.g. 
```
if (true) {
   do sth..
}
```
It's more error-proof.